projects/observability/src/shared/dashboard/widgets/charts/cartesian-widget/cartesian-widget-renderer.component.ts

```diff
@@ -72,14 +74,15 @@ export class CartesianWidgetRendererComponent<TSeriesInterval, TData> extends In
     return this.model.getDataFetcher().pipe(
       tap(fetcher => {
         this.fetcher = fetcher;
-        const defaultInterval = this.model.defaultInterval?.getDuration();
+        const defaultInterval = this.getDefaultInterval();
         const intervalOptions = this.buildIntervalOptions();
 
         if (this.intervalSupported()) {
           this.selectedInterval = this.getBestIntervalMatch(intervalOptions, this.selectedInterval ?? defaultInterval);
           this.intervalOptions = intervalOptions; // The only thing this flag controls is whether options are available (and thus, the selector)
         } else {
-          this.selectedInterval = this.getBestIntervalMatch(intervalOptions, defaultInterval);
+          // TODO: why do we need to set this if interval is not supported.
```

Comment:
See comment above. It's poorly named, but this is really switching on whether the interval is selectable rather than supported.

Reply:
Yeah, so I want to keep it undefined if the interval is not supported.

Reply:
Maybe just terminology; there _is_ an interval, it's just not selectable. Is it just no longer needed if not selectable? If so, is there any harm in sending it in?